File: app/src/main/java/com/example/whattodo/ActivityCreditCard.java
package com.example.whattodo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class ActivityCreditCard extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_creditcard);
        getSupportFragmentManager().beginTransaction().replace(R.id.frame_creditcard, new CreditCardFragment()).commit();
    }
}

File: app/src/main/java/com/example/whattodo/ActivityCreditCard.kt
package com.example.whattodo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class ActivityCreditCard : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_creditcard)
        supportFragmentManager.beginTransaction().replace(R.id.frame_creditcard, CreditCardFragment()).commit()
    }
}